README.md

```diff
@@ -99,6 +99,87 @@ Add this to your `.pre-commit-config.yaml`
       use `args: [--markdown-linebreak-ext=md]` (or other extensions used
       by your markdownfiles).  If for some reason you want to treat all files
       as markdown, use `--markdown-linebreak-ext=*`.
+- `check-xml-encoding` - Checks that xml files have required encoding as first line.
+    - To specify desired encoding use `args: [--encoding]`. Example:
+    `args: [
+            --encoding, '<?xml version="1.0" encoding="utf-8"?>',
+          ]`
+- `check-location` - Checks that specified files are located inside specified directories.
+    - To specify desired encoding use: `args: [--directories, --files]`. Example:
+    `args: [
+          --directories, '^[^\/]+$',
+          --directories, '.*\/controllers$',
+          --directories, '.*\/data$',
+          --directories, '.*\/i18n$',
+          --directories, '.*\/models$',
+          --directories, '.*\/report$',
+          --directories, '.*\/security$',
+          --directories, '.*\/static\/img$',
+          --directories, '.*\/static\/lib\/external_lib$',
+          --directories, '.*\/static\/src\/js$',
+          --directories, '.*\/static\/src\/css$',
+          --directories, '.*\/static\/src\/less$',
+          --directories, '.*\/static\/src\/xml$',
+          --directories, '.*\/static\/tests$',
+          --directories, '.*[^\/static]\/tests$',
+          --directories, '.*\/views$',
+          --directories, '.*\/wizard$',
+          --files, '__init__.py$ __openerp__.py$',
+          --files, '__init__.py$ .*\.py$ main.py$',
+          --files, '.*_data\.xml$ .*_demo\.xml$',
+          --files, '.*\.po$',
+          --files, '__init__.py$ .*\.py$',
+          --files, '__init__.py$ .*\.py$ .*_views\.xml$ .*_reports\.xml$ .*_templates\.xml$',
+          --files, 'ir.model.access.csv$ .*_security\.xml$',
+          --files, '.*\.png$ .*\.jpg$',
+          --files, '.*\$',
+          --files, '.*\.js$',
+          --files, '.*\.css$',
+          --files, '.*\.less$',
+          --files, '.*\.xml$',
+          --files, '.*\.tour\.js$',
+          --files, '__init__.py$ test_.*\.py',
+          --files, '.*_templates\.xml$ .*_views\.xml$',
+          --files, '.*\.py$ .*_views\.xml$',
+        ]`
```

Comment:
there's a lot going on here, and it might not be obvious from a reader what this means (it wasn't to me!)

Also, for multiline fixed width text, use triple-backticks:

```markdown

    ```yaml
    # yaml here
    ```
```